pycheribuild/projects/run_qemu.py

```diff
@@ -256,9 +259,17 @@ def gdb_command(main_binary, breakpoint=None, extra_binary=None) -> str:
                 self.info("If you would like to debug a userspace program (e.g. sbin/init):")
                 self.info("\t", coloured(AnsiColour.red, gdb_command(self.rootfs_path / "sbin/init", "main", path_to_kernel)), sep="")
             self.info("Launching QEMU in suspended state...")
+
+            if self.config.debug_kernel:
+                server = libtmux.Server()
+                session = server.list_sessions()[0] # hopefully
```

Comment:
What if I have multiple sessions? What if there are no sessions? Both should be handled without doing the wrong thing or crashing with an exception.